hknweb/academics/models/base_models.py

```diff
@@ -0,0 +1,5 @@
+from django.db import models
+
+class AcademicEntity(models.Model):
+    created_at = models.DateTimeField()
+    updated_at = models.DateTimeField()

```

Comment:
I'm not sold on the need for a base class just for time fields. Any features we might actually anticipate?

Reply:
Good question, I should have made a point of this in the commit description. Eventually this AcademicEntity class will contain entity-type-agnostic functions like `merge` and `separate`, along with several methods that those two functions will require e.g. `get_value_attributes` and `get_reference_attributes`, etc. There's not documentation about these functions as of this moment, but when there is, it can be found in the [Database operations design doc](https://docs.google.com/document/d/1Xa56DlWdIGqI_nUDE9MkZKB12zg2vpR9dpLgEFzzyVc/edit).